packages/web-reporter-ui/src/components/Chart.tsx

```diff
@@ -107,6 +137,7 @@ export const Chart = ({
       },
       annotations: {
         xaxis: videoEnabled ? [getVideoCurrentTimeAnnotation()] : [],
+        yaxis: getAnnotationInterval(annotationIntervalList),
```

Comment:
eslint is saying you need to add the `annotationIntervalList` to the array of deps

Reply:
The eslint config and prettier is a little weird isn't it ? It does the prettier only after the push...